### PR TITLE
Intercom Resprite and Tweaks

### PR DESCRIPTION
**Pull request by DrDiasyl**
## About The Pull Request

Changes the sprites of the Station Intercom, Command Intercom, and Prison Intercom to the next ones:
new intercom
new command intercom
prison intercom

Now Station Intercom and Command Intercom descriptions are changed to tell more about their real functionality.
Command Intercom now has High-Volume settings like heads headsets.

There is now a memo with frequencies of every channel on the station, the memo is placed near the Command Intercom and in the Telecommunications Room.
## Why It's Good For The Game

Intercoms are part of the "old wall-mount" sprites that were never touched, besides the brief addition of overlays.
Replacing them to design more familiar of other wall-mounts will make it less ancient.

Making the Command Intercom and Prison Intercom look unique ensures players (especially new ones) can distinguish them by just glancing without needing to examine them.
The Command Intercom having a High-Volume setting makes it even more unique as an item that is only met in one room at the station, also makes any Command orders given through an intercom (during Telecomunnications outages) more noticeable.

The description change gives more useful information about Station Intercom and Command Intercom. Now players (especially new ones) can understand that Station Intercoms are usable during telecommunications outages, and Command Intercoms can be set to any frequency.

The memo will help players (especially new ones) understand what numbers they should input to access certain channels using Command Intercom, as right now, the only way to find out the frequencies is through wiki or directly checking servers.

## Changelog
:cl:
qol: The Command intercom now has a High-Volume setting like command headsets
qol: A memo telling frequencies of station radio channels are now present near the Command intercom and T-Comms room
image: Station, Command, and Prison intercoms have received new sprites
spellcheck: Station and Command intercom descriptions have been changed to tell about their functionality
/:cl:

### CODE REVIEW

**Comment by mc-oofert** [comment]
i dont think this is any better than the prior sprites

**Comment by Serxule** [comment]
this is a downgrade honestly, its slightly better in quality but much less clear in what it is and what it does, its also very bland and would be very hard to notice compared to current ones

**Comment by DrDiasyl** [comment]
> this is a downgrade honestly, its slightly better in quality but much less clear in what it is and what it does, its also very bland and would be very hard to notice compared to current ones

The old intercom sprite had much more colour shifting in pixels, as such it was more of a pixel blob than this one.
I can't agree that it became less clear, but I can see how some people might notice them on the walls. But I guess the red buttons and the blue screen must give it away.

**Comment by DrDiasyl** [comment]
> > this is a downgrade honestly, its slightly better in quality but much less clear in what it is and what it does, its also very bland and would be very hard to notice compared to current ones
> 
> The old intercom sprite had much more colour shifting in pixels, as such it was more of a pixel blob than this one. I can't agree that it became less clear, but I can see how some people might not notice them on the walls. But I guess the red buttons and the blue screen must give it away.

**Comment by EuSouAFazer** [comment]
Good change, looks a lot better. Don't let the soul argument get to you

**Comment by CoiledLamb** [comment]
> > this is a downgrade honestly, its slightly better in quality but much less clear in what it is and what it does, its also very bland and would be very hard to notice compared to current ones
> 
> The old intercom sprite had much more colour shifting in pixels, as such it was more of a pixel blob than this one. I can't agree that it became less clear, but I can see how some people might not notice them on the walls. But I guess the red buttons and the blue screen must give it away.

if you're conflicted about it, it might be worth posting some images of how it looks in game?
I think they're fine though, personally. might be worth sampling colors from the wallening ver if you didn't make your own palette

File: code/modules/paperwork/paper_premade.dm [hunk 1]
```diff
@@ -72,6 +72,19 @@
...
+	default_raw_text = {"Please remember the frequencies of each radio channel used on station:
+	* AI Private - 144.7
+	* Command - 135.3
+	* Common - 145.9
+	* Engineering - 135.7
+	* Medical - 135.5
+	* Science - 135.1
+	* Security - 135.9
+	* Service - 134.9
+	* Supply - 134.7"}
```
Comment (Mothblocks) [inline code/modules/paperwork/paper_premade.dm:86]: Indent up one

**Comment by Mothblocks** [comment]
wut

https://github.com/tgstation/tgstation/assets/35135081/728868f9-66d8-41e1-80b5-2cda1b3754be

**Comment by Mothblocks** [comment]
also plz dont add new "-p" sprites especially since you have the variable now, give it a complete name "powered? unpowered?"

**Comment by Mothblocks** [comment]
also lets see it in game on some walls since someone said they wonder if it will look bland on them

**Comment by Sealed101** [comment]
> wut

diffbot has been freaking out for a few weeks like this, iirc aa was informed at some point

**Comment by DrDiasyl** [comment]
> also lets see it in game on some walls since someone said they wonder if it will look bland on them

image

**Comment by DrDiasyl** [comment]
oh great wrong one, here is the right one
Intercoms

**Comment by CoiledLamb** [comment]
damn that palette does flow directly into the wall in places

**Comment by DATA-xPUNGED** [comment]
I'm gonna be real with u didn't knew there were specific command intercoms

**Comment by EricZilla** [comment]
Love this as an independent sprite but it really needs to not blend with iron walls, maybe tone all the grey up a bit?

**Comment by Serxule** [comment]
> oh great wrong one, here is the right one

is it not multidirectional or

**Comment by CoiledLamb** [comment]
> > oh great wrong one, here is the right one
> 
> is it not multidirectional or

current one is not multidirectional, you might be thinking of the wallening one.
I don't think multidirectional wallmounts have a great chance of approval either

**Review by Wallemations — COMMENTED** [review]
Why are you making these individual icons instead of just adding overlays to a normal sprite?

**Comment by DrDiasyl** [comment]
> Why are you making these individual icons instead of just adding overlays to a normal sprite?

Good question. Just intercoms already have overlays, but I _think_ overlay bloat doesn't exist.

**Comment by Serxule** [comment]
> > > oh great wrong one, here is the right one
> > 
> > 
> > is it not multidirectional or
> 
> current one is not multidirectional, you might be thinking of the wallening one. I don't think multidirectional wallmounts have a great chance of approval either

i think what looks off to me is that its not centered anymore

also, i'd like to add, this one looks less like a radio by using blue, green/orange are much more iconic and recognizeable as a "radio" color

**Comment by DrDiasyl** [comment]
> > > > oh great wrong one, here is the right one
> > > 
> > > 
> > > is it not multidirectional or
> > 
> > 
> > current one is not multidirectional, you might be thinking of the wallening one. I don't think multidirectional wallmounts have a great chance of approval either
> 
> i think what looks off to me is that its much more offcentered
> 
> also, i'd like to add, this one looks less like a radio by using blue, green/orange are much more iconic and recognizeable as a "radio" color

Surprisingly the old sprite was the one off-centred, I actually have centred it. Also, the original Intercom sprite had a blue screen, hence this one does too.

**Comment by DrDiasyl** [comment]
> The sprites themselves look alright, but some in-game screenshots would help.

I posted some in the comments

**Comment by epochayur** [comment]
Blends in with the walls WAY too much

**Comment by HalfPunch** [comment]
Now the intercoms look like grandfathers radio
image

**Comment by DrDiasyl** [comment]
> Blends in with the walls WAY too much

Just like the old one's though.

**Comment by CoiledLamb** [comment]
> > Blends in with the walls WAY too much
> 
> Just like the old one's though.

is a fixable issue. what is the point of a resprite if it has the same issues as what it's respriting?

**Review by Wallemations — CHANGES_REQUESTED** [review]
You have really zoomed in screenshots, but can we just get a game screenshot with a character in there? Maybe just load up some map and stand in the middle of a hallway & take a 6x6 tile screenshot or something to get the whole picture.

I wanna see how it looks in conjunction with other objects rather than just the one.

**Comment by DrDiasyl** [comment]
> You have really zoomed in screenshots, but can we just get a game screenshot with a character in there? Maybe just load up some map and stand in the middle of a hallway & take a 6x6 tile screenshot or something to get the whole picture.
> 
> I wanna see how it looks in conjunction with other objects rather than just the one.

@Wallemations here you go
image
image
image
image

**Comment by Wallemations** [comment]
I like them on the north and south but nudge them a little more in for the side walls, it looks like they're peeking a pixel or three out

**Review by Ghommie — CHANGES_REQUESTED** [review]
Sorry for the late review. Fix the intercom_receive icon state and the conflicts, and you're good to go. Also what Wallemations said.

**Comment by DrDiasyl** [comment]
> Sorry for the late review. Fix the intercom_receive icon state and the conflicts, and you're good to go. Also what Wallemations said.

All done @Ghommie

**Pull request merged**